index.bs

```diff
@@ -3925,97 +3925,43 @@ Transitioning to Proposed Recommendation</h4>
 			<em class="rfc2119">may</em> have removed features
 			identified in the [=Candidate Recommendation Snapshot=] document as [=at risk=]
 			without republishing the specification as a [=Candidate Recommendation Snapshot=].
-	</ul>
-
-	The [=Team=]:
-
-	<ul>
-		<li>
-			<em class="rfc2119">must</em> announce the publication of a [=Proposed Recommendation=]
-			to the <a href="#AC">Advisory Committee</a>,
-			and <em class="rfc2119">must</em> begin an [=Advisory Committee Review=]
-			on the question of whether the specification is appropriate to [=publish=] as a [=W3C Recommendation=].
 
 		<li>
-			<em class="rfc2119">may</em> approve a [=Proposed Recommendation=]
-			with minimal <a href="#implementation-experience">implementation experience</a>
-			where there is a compelling reason to do so.
-			In such a case, the [=Team=] <em class="rfc2119">must</em> explain the reasons for that decision,
-			and that information <em class=rfc2119>must</em> be included in the [=Call for Review=]
-			proposing advancement to [=W3C Recommendation=].
+			<em class="rfc2119">must</em> identify, in the document, where errata are tracked.
 	</ul>
 
-	Since a [=W3C Recommendation=] <em class="rfc2119">must not</em> include any [=substantive changes=]
-	from the [=Proposed Recommendation=] it is based on,
-	to make any [=substantive change=] to a [=Proposed Recommendation=]
-	the [=Working Group=] <em class="rfc2119">must</em> return the specification to [=Candidate Recommendation=]
-	or [=Working Draft=].
-
-	<p id="expandable-rec">
-	A [=Proposed Recommendation=] may identify itself
-	as intending to <dfn>allow new features</dfn>
-	(<a href="#correction-classes">class 4 changes</a>)
-	after its initial publication as a [=Recommendation=],
-	as described in [[#revised-rec-features]].
-	Such an allowance cannot be added
-	to a [=technical report=] previously published as a [=Recommendation=]
-	that did not allow such changes.
-
-	Possible Next Steps:
-
-	<ul>
-		<li>
-			Return to <a href="#revising-wd">Working Draft</a>
-
-		<li>
-			Return to <a href="#transition-cr">Candidate Recommendation</a>
-
-		<li>
-			<a href="#transition-rec">Recommendation status</a>,
-			(the expected next step).
-
-		<li>
-			<a href="#abandon-draft">Discontinued Draft</a></li>
-	</ul>
-
-	[=Advisory Committee representatives=] <em class="rfc2119">may</em> initiate an [=Advisory Committee Appeal=]
-	of the decision to advance the technical report.
+	Additionally,
+	if the document has previously been published as a [=W3C Recommendation=], the [=Working Group=]
+	<em class=rfc2119>must not</em> include any <a href="#correction-classes">class 4 change</a>
+	to that publication
+	unless it was explicitly marked as [=allowing new features=],
+	and <em class=rfc2119>must not</em> include any such marking
+	if not already present.
```

Comment:
This also appears to block addition of features even via the Rec -> CR -> Rec process, which I think is unintended. The problem is that "has previously been published" means "ever at any time in the past" rather than "just the most recently published version".

Reply:
It does block that, but it is in fact intended, and predates this Pull Request. See the more detailed comment I left at https://github.com/w3c/process/pull/868#discussion_r1656535961. We can debate whether this is a good idea, but since it predates this Pull Request, I'd request you open a separate issue about that.